base/src/main/java/com/smartdevicelink/proxy/rpc/AppCapability.java

```diff
@@ -0,0 +1,49 @@
+package com.smartdevicelink.proxy.rpc;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.proxy.RPCStruct;
+import com.smartdevicelink.proxy.rpc.enums.AppCapabilityType;
+
+import java.util.Hashtable;
+
+public class AppCapability extends RPCStruct {
+    public static final String KEY_APP_CAPABILITY_TYPE = "appCapabilityType";
+    public static final String KEY_VIDEO_STREAMING_CAPABILITY = "videoStreamingCapability";
+
+    public AppCapability() {
+    }
+
+    /**
+     * Constructs a new AppCapability object indicated by the Hashtable
+     * parameter
+     *
+     * @param hash hashtable filled with params to create an instance of this RPC
+     *             The hash table to use
+     */
+    public AppCapability(Hashtable<String, Object> hash) {
+        super(hash);
+    }
+
+    public AppCapability(@NonNull VideoStreamingCapability capability, AppCapabilityType appCapabilityType) {
+        this();
```

Comment:
> `VideoStreamingCapability capability` is not mandatory, so you do not need to initialize it in this constructor.

@kostyaBoss I still see `VideoStreamingCapability capability` being passed into the constructor on line 29.  You don't need it, since it is not a mandatory field.